cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -29,7 +29,9 @@ set(SNAPPY_VERSION "1.1.3")
 set(BROTLI_VERSION "v0.6.0")
 set(LZ4_VERSION "1.7.5")
 set(ZSTD_VERSION "1.2.0")
+set(PROTOBUF_VERSION "2.6.0")
 set(GRPC_VERSION "94582910ad7f82ad447ecc72e6548cb669e4f7a9") # v1.6.5
+set(ORC_VERSION "cf00b67795717ab3eb04e950780ed6d104109017")
```

Comment:
Does this need a version of trunk, or can we use a released version of ORC?

Reply:
Needs a version of trunk (needed a bunch of fixes to their build system). This was master when I started this PR, but could probably update to master again.